src/iOS.Core/Controllers/LoginAddViewController.cs

```diff
@@ -113,6 +135,42 @@ public override void ViewDidLoad()
             folderNames.Insert(0, AppResources.FolderNone);
             FolderCell.Items = folderNames;
 
+            _personalOwnershipPolicyApplies = _policyService.PolicyAppliesToUser(PolicyType.PersonalOwnership).GetAwaiter().GetResult();
+            var index = 0;
+            if (_personalOwnershipPolicyApplies)
```

Comment:
🎨 This method is pretty huge, IMO personal ownership logic should be extracted to a different method we can call here so it doesn't make this method even bigger than it is.